src/copy.rs

```diff
@@ -12,6 +12,7 @@ pub enum MemoryCopyError {
     #[fail(display = "Failed to copy memory address {:x}", _0)] Io(usize, #[cause] std::io::Error),
     #[fail(display = "Process isn't running")] ProcessEnded,
     #[fail(display = "Other")] Other,
+    #[fail(display = "Too much memory requested when copying")] InvalidLength,
```

Comment:
I'd make the name be `RequestTooLarge` or something more indicative of the actual issue. Also, take a `usize` parameter and include it in the display message.